components/section-home/index.vue

```diff
@@ -4,16 +4,38 @@
 		<home-title style="grid-area: title" />
 		<home-description style="grid-area: description" />
 	</section>
+	<section class="section-earnings">
+		<h1> Current APY 5.5% </h1> <!-- This of course can be pulled from where ever. -->
+	</section>
 </template>
 
 <script setup lang="ts">
 import homeTezzomboiz from './home-tezzomboiz'
 import homeTitle from './home-title'
 import homeDescription from './home-description'
+
 </script>
 
 <style lang="scss" scoped>
 @import 'include-media';
+
+.section-earnings {
+	text-align: center;
+	margin-top: 10%;
+}
+
+h1 {
+		color: white;
+		font-size: clamp(2rem, 10vw, 4rem);
+
+		img {
+			display: inline-block;
+			vertical-align: text-bottom;
+			height: clamp(3rem, 15vw, 5rem);
+			vertical-align: middle;
+		}
+	}
+
```

Comment:
The CSS margins for mobile have disappeared but I'm not quite sure what's the cause.

![image](https://user-images.githubusercontent.com/294987/202042835-16c9b209-60fd-40a3-b9e3-0349a74555b6.png)